hydroflow_plus_std/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "hydroflow_plus_std"
+publish = true
+version = "0.1.0"
```

Comment:
When we release, we should probably match this with the rest of the versions, would be 0.11